django-env/bin/EXIF.py

```diff
@@ -0,0 +1,129 @@
+#!/Users/latoniamerticasheppardwalker/Documents/JTC_Projects/wow-project/Inheritance/django-env/bin/python3.9
+# -*- coding: utf-8 -*-
+#
+#
+# Library to extract Exif information from digital camera image files.
+# https://github.com/ianare/exif-py
+#
+#
+# Copyright (c) 2002-2007 Gene Cash
+# Copyright (c) 2007-2020 Ianaré Sévi and contributors
+#
+# See LICENSE.txt file for licensing information
```

Comment:
the license for this project has some requirements that need to be honored: https://github.com/ianare/exif-py/blob/develop/LICENSE.txt

if you're going to use this library, i think you're required to include its license in your repo. i'm not actually sure the best way to do this...